yang-semver/draft-ietf-netmod-yang-semver.xml

```diff
@@ -191,24 +199,28 @@
 <li>'Y' is the MINOR version.  Changes in the MINOR version number
        indicate changes that are backwards-compatible to versions with
        the same MAJOR version number, but a lower MINOR version number
-       and no PATCH "_compatible" or "_non_compatible" modifier.</li>
+       and no "_compatible" or "_non_compatible" modifier.</li>
+<li>'Z' is the PATCH version. Changes in the PATCH
+       version number can indicate an editorial change to the YANG artifact.
+       In conjunction with the '_COMPAT' modifier (see below) changes to 'Z'
+       may indicate a larger module change.
```

Comment:
not a huge fan of "a larger module change". We ended up using the term "impactful" in my recent Module Versioning updates, so perhaps "may indicate a more impactful module change" ?

Reply:
I'm not a fan, either.  But I don't like the word "impactful."  What about, "a more substantive module change"?

Reply:
I'm not loving that either, but don't care too much so if you're keen on substantive then it is probably clear enough.

Reply:
Okay.  I'll make the change.